src/shared/lib/utils/survey.ts

```diff
@@ -0,0 +1,10 @@
+export const invertColor = (hex: string) => {
+  const hexColor = hex.replace('#', '');
+  const r = parseInt(hexColor.substring(0, 2), 16);
+  const g = parseInt(hexColor.substring(2, 4), 16);
+  const b = parseInt(hexColor.substring(4, 6), 16);
+  const yiq = (r * 299 + g * 587 + b * 114) / 1000;
```

Comment:
can we rename yiq, not clear the goal of the variable, also 299, 587 looks like magic numbers. Can we create constants for them?